components/LinkCustom/LinkCustom.tsx

```diff
@@ -40,3 +40,4 @@ const LinkCustom = ({
 };
 
 export default LinkCustom;
+
```

Comment:
NIT: How come there's an extra line here?

Reply:
I didn't want the first extra line to feel lonely ... both gone now.